### ISSUE TITLE
Cannot build the compiler on Windows: build.bat suddenly quits

### ISSUE THREAD

**Issue by anta40** [open-watcom/open-watcom-v2#266]
I have a spare laptop (running 64-bit Win 8), which is used to build the 32-bit Open Watcom. 
Of course, I already have the Open Watcom V2 x86 installed.

I cannot build the latest version from git (current commit ID is a710c92). Every time I run build.bat (which usually will start the building process, it just quits).

Here are the steps to reproduce the problem:
1. Put the source code in C:\ow\
2. Run the Open Watcom Build Environment, and type:
```
set PATH=%PATH%;C:\Windows\System32;"C:\Program Files (x86)\
gs\gs9.19\bin";"C:\Program Files (x86)\gs\gs9.19\lib"
```
3. Do some changes on setvars.bat:
```
set OWROOT=c:\ow
set OWDOSBOX=C:\DOSBox\DOSBox.exe
```
4. Run owconfig.bat
5. Run servars.bat
6. Run build.bat

Nothing happens.

**Comment by jmalak** [comment]
It is strange. It looks like wmake or builder build failed for some reason.
Look into OW tree root for file bootx.log.
There is log from pre-build wmake and builder tools.

**Comment by jmalak** [comment]
It should be fixed by commit 6631bfe572dc8e525e39bee6732e30b507a7bb8c
